### DISCUSSION TITLE
Errors on page ls

### DISCUSSION THREAD

**Discussion by fipachu** [tldr-pages/tldr#10856] [General]
A few of the examples on page [ls](https://tldr.inbrowser.app/pages/common/ls) say they list all files, but they don't include "-a", and therefore don't actually list all files.
![image](https://github.com/tldr-pages/tldr/assets/80906036/3b567376-b329-42be-b66a-c01fb17f6d85)

I didn't know how to report this so I'm starting a discussion, I hope it's ok.

**Comment by kbdharun** [comment]
Hi, thanks for starting the discussion. It is intentional as the flag `-a` is only required for displaying hidden files (files starting with a period), which are meant to be hidden from the user.  (And in most cases it isn't recommended for the general user to see it as they might delete the file unintentionally.) [Developers might prefer to see the hidden files so for them the page includes 2 examples with the flag while explaining other essential flags]
